import "response.proto";
import "serverinfo_replay.proto";

message Response_ReplayList {
	extend Response {
		optional Response_ReplayList ext = 1100;
	}
	repeated ServerInfo_Replay replay_list = 1;
}
